data/lang/core/en.json

```diff
@@ -1801,7 +1801,7 @@
   },
   "WITH_A": {
```

Comment:
This could be changed to `"WITH"` but I see no good reason to do this at this point if the code is going to be rewritten anyway.